### DISCUSSION TITLE
Ssh tunnel connection

### DISCUSSION THREAD

**Discussion by Ez-Bruno** [apache/doris#22478] [A - General / Q&A]
I dont know how to connect postgres db to apache doris instance with ssh tunnel integration.

Hi, im trying to connect apache doris to postgres with catalogs. Im using this documentation https://doris.apache.org/docs/dev/lakehouse/multi-catalog/jdbc, and work with normal database connection, but i dont know how to do the same with ssh tunnel. 
First i think is use another driver with ssh included like https://github.com/cactuslab/jdbc-sshj/tree/master, but i dont found documentation to use custom drivers in apache.

**Comment by LemonLiTree** [comment]
only support catalog
